src/iohandler/mem_io_handler.cc

```diff
@@ -35,7 +35,9 @@
 #include <cstdlib>
 #include <cstring>
 #include <ctime>
+#ifndef USING_NPUPNP
```

Comment:
Instead of wrapping with ifdef it's better to just remove unused header.
Maybe there are other similar places...

Reply:
All of these headers seem to be unnecessary...

Removed.